client/my-sites/checkout/src/components/wp-checkout-order-review.tsx

```diff
@@ -22,7 +23,6 @@ import type { OnChangeItemVariant } from './item-variation-picker';
 import type { CouponFieldStateProps } from '../hooks/use-coupon-field-state';
 import type { SiteDetails } from '@automattic/data-stores';
 import type { ResponseCart, RemoveProductFromCart, CouponStatus } from '@automattic/shopping-cart';
-
```

Comment:
It's not that important but I noticed you removed the new line after the imports in this file and several others. Was that intentional? I find having a newline between the imports and the rest of the file makes it easier to read.